apps/address-service/package.json

```diff
@@ -48,7 +48,7 @@
     "@welldone-software/why-did-you-render": "^5.0.0-alpha.1",
     "@zeit/next-css": "^1.0.1",
     "@zeit/next-less": "^1.0.1",
-    "antd": "^4.8.2",
+    "antd": "^4.24.12",
```

Comment:
Do you really need `antd` in `address-service`?)

Reply:
:) probably